modules/nw-networkpolicy-concept.adoc

```diff
@@ -0,0 +1,47 @@
+[id="nw-networkpolicy-audit-concept_{context}"]
+= Network policy audit logging
+
+The OVN-Kubernetes cluster network provider uses OVN (Open Virtual Network) ACLs to manage network policy. Audit logging exposes allow and deny ACL events.
+
+You can configure where network policy audit logs are sent, such as to a syslog server or to a specified Unix Domain Socket. Regardless of any additional configuration, an audit log is always saved to `/var/log/ovn/acl-audit-log.log` on each OVN-Kubernetes pod in the cluster.
+
+Network policy audit logging is enabled per namespace by annotating the namespace with the `k8s.ovn.org/acl-logging` key as in the following example:
+
+.Example namespace annotation
+[source,yaml]
+----
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: example1
+  annotations:
+    k8s.ovn.org/acl-logging: |-
+      {
+        "deny": "info",
+        "allow": "info"
+      }
+----
+
+The logging format used is syslog compatible as defined by RFC5424. The syslog facility is configurable and defaults to `local0`. An example log entry might resemble the following:
```

Comment:
The first sentence reads a little awkwardly.

Obligatory sugg: "The logging format is compatible with syslog as defined by RFC5424."